app/views/posts/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if @posts.present? %>
+<%= render(partial: 'users/user_card', locals: {user: @posts.first.user}) %>
+<% else %>
+This user doesn't have any post yet!
+<% end %>
+
+<% @posts.each do |post| %>
+  <%= render partial: 'post_card', locals: { post: post, comments: post.recent_five_comments} %>
+<% end %>
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/179355010-88c74917-c9a7-4c25-ab00-ff6cac433c17.png)

- [ ] Kindly fix the error on this route